components/tabs/tabs.js

```diff
@@ -340,22 +348,22 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(RtlMixin(FocusVisiblePolyf
 							<d2l-icon icon="tier1:chevron-left"></d2l-icon>
 						</button>
 					</div>
-					<span class="d2l-tabs-focus-start" @focus="${this._handleFocusStart}" tabindex="0"></span>
```

Comment:
I think there is some CSS for these focus-trap classes that can also be removed.